src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -14,14 +17,29 @@
 )
 
 from databricks.labs.ucx.config import WorkspaceConfig
-from databricks.labs.ucx.framework.crawlers import SqlBackend, StatementExecutionBackend
+from databricks.labs.ucx.framework.crawlers import (
+    CrawlerBase,
+    SqlBackend,
+    StatementExecutionBackend,
+)
 from databricks.labs.ucx.hive_metastore import TablesCrawler
 from databricks.labs.ucx.hive_metastore.mapping import Rule, TableMapping
 from databricks.labs.ucx.hive_metastore.tables import MigrationCount, Table, What
 
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class TableMigrationStatus:
+    src_workspace_id: str
```

Comment:
on this level, we have nothing to do with a `workspace_id`

Reply:
OK

Reply:
Removed